.gitignore

```diff
@@ -1,48 +1,7 @@
-# Built style type definitions
```

Comment:
These were all duplicated for some reason, this is just removing those duplicates.